test/evergreen.yml

```diff
@@ -132,32 +132,76 @@ functions:
         set -o errexit
         set -o verbose
         for i in $(seq ${times|1}); do
-          ./t -1 -c ${config|../../../test/format/CONFIG.stress} ${extra_args|}
+          ./t -1 -c ${config|../../../test/format/CONFIG.stress} ${extra_args|} 2>&1
         done
   "many dbs test":
     command: shell.exec
-    parms:
+    params:
       working_dir: "wiredtiger/build_posix/test/manydbs"
       script: |
         set -o errexit
         set -o verbose
-        ${test_env_vars|} ./t ${many_db_args|}
+        ${test_env_vars|} ./t ${many_db_args|} 2>&1
   "thread test":
     command: shell.exec
-    parms:
+    params:
       working_dir: "wiredtiger/build_posix/test/thread"
       script: |
         set -o errexit
         set -o verbose
-        ${test_env_vars|} ./t ${thread_test_args|}
+        ${test_env_vars|} ./t ${thread_test_args|} 2>&1
   "random abort test":
     command: shell.exec
-    parms:
+    params:
+      working_dir: "wiredtiger/build_posix/test/csuite"
+      script: |
+        set -o errexit
+        set -o verbose
+        ${test_env_vars|} ./test_random_abort ${random_abort_args|} 2>&1
+  "timestamp abort test":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/build_posix/test/csuite"
+      script: |
+        set -o errexit
+        set -o verbose
+        ${test_env_vars|} ./test_timestamp_abort ${timestamp_abort_args|} 2>&1
+  "truncated log test":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/build_posix/test/csuite"
+      script: |
+        set -o errexit
+        set -o verbose
+        ${test_env_vars|} ./test_truncated_log ${truncated_log_args|} 2>&1
+  "recovery stress test script":
+    command: shell.exec
+    params:
       working_dir: "wiredtiger/build_posix/test/csuite"
       script: |
         set -o errexit
         set -o verbose
-        ${test_env_vars|} ./test_random_abort ${random_abort_args|}
+
+        for i in $(seq ${times|1}); do
+          # Run the various combinations of args.  Let time and threads be random.
+          # Run current version with write-no-sync txns.
+          ${test_env_vars|} ./test_random_abort 2>&1
+          ${test_env_vars|} ./test_timestamp_abort 2>&1
+
+          # Current version with memory-based txns (MongoDB usage).
+          ${test_env_vars|} ./test_random_abort -m 2>&1
+          ${test_env_vars|} ./test_timestamp_abort -m 2>&1
+
+          # V1 log compatibility mode with write-no-sync txns.
+          ${test_env_vars|} ./test_random_abort -C 2>&1
+          ${test_env_vars|} ./test_timestamp_abort -C 2>&1
+
+          # V1 log compatibility mode with memory-based txns.
+          ${test_env_vars|} ./test_random_abort -C -m 2>&1
+          ${test_env_vars|} ./test_timestamp_abort -C -m 2>&1
+
+          ${test_env_vars|} ./test_truncated_log ${truncated_log_args|} 2>&1
```

Comment:
I guess any line of test fails would cause the Evergreen task fails and stops?

Reply:
With `set -o errexit` statement in script, the task should fail if any of the commands shall fail.

Reply:
That means the new task won't be running through 24hrs if any line of test fails in any round?

Reply:
That's correct. The only way out would be to use something like `format.sh` script that can manage spawning job that can potentially fail.

Reply:
Got it